agent.go

```diff
@@ -953,14 +954,15 @@ func (a *Agent) findRemoteCandidate(networkType NetworkType, addr net.Addr) Cand
 func (a *Agent) sendBindingRequest(m *stun.Message, local, remote Candidate) {
 	a.log.Tracef("ping STUN from %s to %s\n", local.String(), remote.String())
 
-	if overflow := len(a.pendingBindingRequests) - (maxPendingBindingRequests - 1); overflow > 0 {
-		a.log.Debugf("Discarded %d pending binding requests, pendingBindingRequests is full", overflow)
-		a.pendingBindingRequests = a.pendingBindingRequests[overflow:]
+	nRemoved := a.invalidatePendingBindingRequests()
+	if nRemoved > 0 {
+		a.log.Tracef("Discarded %d binding requests because they expired", nRemoved)
```

Comment:
Do you want this as a Trace message? It prints how many binding requests were removed due to expiration.

Reply:
I think this is good!